### ISSUE TITLE
Support declaring accounts via the add form.

### ISSUE THREAD

**Issue by mcbloch** [beancount/fava#1176]
There are a few ways this could be done

- Add 4th type in the form UI for Accounts: very explicit.
- Suggest declaring an unrecognized Account when entering a Transaction or Balance with it: more implicit.

**Comment by yagebu** [comment]
Yes, that should be supported. I think more explicit is the way to go here.

**Comment by alycda** [comment]
I'd like to point out that while I was first getting started, the "+" button wasn't working for me. Likely because I had 0 or only 1 account. I understand the entire point of double-entry accounting, but I was discovering how to use the app and didn't get any kind of error message (I would have expected to be able to add accounts via the UI).

**Comment by yagebu** [comment]
> didn't get any kind of error message (I would have expected to be able to add accounts via the UI).

There should be an error message ("Should be one of the declared accounts") on the input field - did that not show?

**Comment by jack9603301** [comment]
What does that mean? I don't quite understand what you're thinking.
